packages/gatsby-theme-docs/README.md

```diff
@@ -1,3 +1,168 @@
-# Gatsby theme for commercetools docs
+# Core Gatsby theme
 
-TBD
+This is the core Gatsby theme for building commercetools documentation websites.
+
+## Getting started
+
+To create a new documentation website you need to install this theme and its peer dependencies:
+
+```
+npx install-peerdeps --dev @commercetools-docs/gatsby-theme-docs
+```
+
+### Choose a path prefix
+
+All commercetools documentation websites are served under `docs.commercetools.com`. In order to make this work, all documentation websites must be bundled for production using a `pathPrefix`. This value determines the URL path where the website is served from.
+
+For example, for the "Custom Applications" website, the path prefix is `/custom-applications`.
+
+The `pathPrefix` is configured in the `gatsby-config.js` file.
+
+### Folder structure
+
+The project structure should contain at least the following files and folders:
+
+```
+├── .eslintrc.yml
+├── gatsby-config.js
+├── package.json
+└── src
+    ├── content
```

Comment:
I thought these are autogenerated on first startup, but that maybe be something that's not ported from the last implementation yet

Reply:
They are, here we only want to document what they are for. Maybe we can mention that some of them are generated on first gatsby start.